libavfilter/vf_dnn_detect.c

```diff
@@ -136,6 +139,96 @@ static int dnn_detect_post_proc(AVFrame *frame, DNNData *output, uint32_t nb, AV
     return 0;
 }
 
+static int dnn_detect_post_proc_tf(AVFrame *frame, DNNData *output, AVFilterContext *filter_ctx)
+{
+    DnnDetectContext *ctx = filter_ctx->priv;
+    int proposal_count;
+    float conf_threshold = ctx->confidence;
+    float *conf_p, *position_p, *label_id_p, x0, y0, x1, y1;
+    int nb_bboxes = 0;
+    AVFrameSideData *sd;
+    AVDetectionBBox *bbox;
+    AVDetectionBBoxHeader *header;
+
+    proposal_count = *(float *)(output[0].data);
+    conf_p         = output[1].data;
```

Comment:
what does 'p' mean?

Reply:
it means pointer

Reply:
does ffmpeg use '_p' for pointer in current code?

Reply:
Yes, there are codes use '_p' for pointer( like vf_nnedi.c and some nasm code), but not only a few.
Or is it better to just use 'conf'?

Reply:
i checked vf_nnedi.c, there are src_p and dst_p, but the reason is that there are already '*src' and '*dst',  I don't see ffmpeg has such style in common.